nextcord/client.py

```diff
@@ -297,6 +301,7 @@ def __init__(
         self._rollout_register_new: bool = options.pop("rollout_register_new", True)
         self._rollout_update_known: bool = options.pop("rollout_update_known", True)
         self._rollout_all_guilds: bool = options.pop("rollout_all_guilds", False)
+        self._rollout_guild_ids: List[int] = options.pop("rollout_guild_ids", list())
```

Comment:
If this is left as a list
```suggestion
        self._rollout_guild_ids: List[int] = options.pop("rollout_guild_ids", [])
```